File: src/main/java/example/MonadicOpTest.java
package example;

public class MonadicOpTest {
    public static int intVar1,intVar2;
    public static float realVar;

    public static void main(String[] args) {
        intVar1 = 1;
        //-1
        System.out.println(-intVar1);

        intVar2 = 2;

        // 1
        System.out.println(-(intVar1 - intVar2));

        // -6
        System.out.println(-5 + 1 - 2);

        // -5
        System.out.println(-5-(1-1));

        // -7
        System.out.println(-5-1-1);

        // 4
        System.out.println(+5-1);

        // 4
        System.out.println(+5 - (+1));

        System.out.println(-5.0);
        float ii = -5.0F;
        float jj = 1.0F;
        int kk = 1;
        //-3.0
        System.out.println(ii + jj + (+kk));
        System.out.println(-5.0 + 1 + (+1.0));

        //-4.0
        System.out.println(-5.0*1+(1.0*1));
    }
}

File: src/test/resources/driver/testPascalRun/monadicOpTest.pas
(*
 * Project: pascal_test
 * User: Lenovo
 * Date: 13/02/2022
 *)
program monadicOpTest(output);
var intVar1,intVar2:Integer;
  realVar:Real;
begin
  intVar1 := 1;
  WriteLn(-intVar1); {-1}
  intVar2 := 2;

  {1}
  WriteLn(-(intVar1 - intVar2)); {1}

  WriteLn(-5 + 1 - 2); {-6}
  WriteLn(-5 - (1 - 1)); {-5}
  WriteLn(-5 - 1 - 1); {-7}

  WriteLn(+5 - 1); {4}
  WriteLn(+5 - (+1)); {4}

  WriteLn(-5.0); {-5.0}
  WriteLn(-5.0+1.0+(+1));{-3.0}
  WriteLn(-5.0+1+(+1.0));{-3.0}
  WriteLn(-5.0*1+(1.0*1)); {-4.0}

end.